<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(80, 80, 194);
background-color: rgb(38, 19, 19);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>339</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Capture path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cap_path">
       <property name="toolTip">
        <string>Set the full capture path. Separate by comma for multiple paths</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>/home/terminalx/Desktop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Choose Auxiliary Camera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cap_auxCam">
       <property name="toolTip">
        <string>choose the auxiliry camera &quot;basler&quot; vs &quot;ids&quot;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>basler</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Queue Size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cap_queueSize">
       <property name="toolTip">
        <string>Size of the queue of fifo buffer</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Bin Size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cap_binSize">
       <property name="toolTip">
        <string>No of images to dumped at one throgh</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Rows_Aux</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="cap_rowAux">
       <property name="toolTip">
        <string>No of Rows for the auxiliary camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1024</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cols_Aux</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="cap_fpsAux">
       <property name="toolTip">
        <string>Frame rate for the auxiliary camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="cap_colsAux">
       <property name="toolTip">
        <string>No of Cloumns for the auxiliary camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1280</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>FrameRate_Aux</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="cap_camID_IDS">
       <property name="toolTip">
        <string>Camera ID for the IDS camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>CamID_IDS</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="cap_rows_endo">
       <property name="toolTip">
        <string>No of Rows for the Endoscopic camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>720</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Rows_Endo</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="cap_cols_endo">
       <property name="toolTip">
        <string>No of cols for the Endoscopic camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1280</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cols_Endo</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_20">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Frame Rate Endo</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="cap_fps_endo">
       <property name="toolTip">
        <string>Frame rate for the Endoscopic camera</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>label_8</zorder>
    <zorder>cap_path</zorder>
    <zorder>label_9</zorder>
    <zorder>cap_queueSize</zorder>
    <zorder>label_10</zorder>
    <zorder>cap_binSize</zorder>
    <zorder>cap_fpsAux</zorder>
    <zorder>label_11</zorder>
    <zorder>cap_colsAux</zorder>
    <zorder>label_12</zorder>
    <zorder>cap_rowAux</zorder>
    <zorder>label_13</zorder>
    <zorder>cap_cols_endo</zorder>
    <zorder>cap_camID_IDS</zorder>
    <zorder>label_18</zorder>
    <zorder>cap_rows_endo</zorder>
    <zorder>label_15</zorder>
    <zorder>label_20</zorder>
    <zorder>cap_fps_endo</zorder>
    <zorder>label_14</zorder>
    <zorder>label_22</zorder>
    <zorder>cap_auxCam</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>169</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Capture Settings</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>339</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_24">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Tracker to be used</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tr_nameTracker">
       <property name="toolTip">
        <string>choose tracker tld or markerBased</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>tld</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_4"/>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>169</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Tracker Settings</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>581</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(5, 5, 14);
gridline-color: rgb(239, 78, 78);
border-color: rgb(115, 219, 50);</string>
   </property>
   <property name="title">
    <string>                       Calibrate Neuro-Endo-Trainer</string>
   </property>
   <widget class="QPushButton" name="btnCalib">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 31, 25);</string>
    </property>
    <property name="text">
     <string>Start Calibration</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>5</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/endo-calib_0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="serialPortBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 53, 53);</string>
    </property>
    <property name="title">
     <string>Select Serial Port</string>
    </property>
    <widget class="QComboBox" name="serialPortInfoListBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="parametersBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>270</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Parameters</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="baudRateLabel">
       <property name="text">
        <string>Baud Rate:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="baudRateBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dataBitsLabel">
       <property name="text">
        <string>Data bits:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dataBitsBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="stopBitsBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="parityBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="stopBitsLabel">
       <property name="text">
        <string>Stop bits:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="flowControlBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="flowControlLabel">
       <property name="text">
        <string>Flow control:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="parityLabel">
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
